examples/platform/silabs/SiWx917/BUILD.gn

```diff
@@ -114,46 +113,9 @@ source_set("siwx917-matter-shell") {
   }
 }
 
-config("attestation-credentials-config") {
-  include_dirs = [ "${chip_root}" ]
-}
-
-source_set("siwx917-attestation-credentials") {
-  sources = [
-    "${silabs_common_plat_dir}/SilabsDeviceAttestationCreds.cpp",
-    "${silabs_common_plat_dir}/SilabsDeviceAttestationCreds.h",
-  ]
-
-  public_deps = [
-    "${chip_root}/src/credentials",
-    "${chip_root}/src/platform:platform_base",
-  ]
-
-  public_configs = [ ":attestation-credentials-config" ]
-}
-
-source_set("silabs-factory-data-provider") {
-  sources = [
-    "${silabs_common_plat_dir}/SilabsDeviceDataProvider.cpp",
-    "${silabs_common_plat_dir}/SilabsDeviceDataProvider.h",
-  ]
-
-  public_deps = [
-    "${chip_root}/src/credentials",
-    "${chip_root}/src/lib/support",
-    "${chip_root}/src/platform:platform_base",
-    "${chip_root}/src/setup_payload",
-  ]
-
-  public_configs = [ ":siwx917-common-config" ]
-
-  if (sl_enable_test_event_trigger) {
-    public_configs += [ ":test-event-trigger-config" ]
-  }
-}
-
 config("siwx917-common-config") {
   defines = [ "OTA_PERIODIC_TIMEOUT=${ota_periodic_query_timeout_sec}" ]
+  libs = [ "${sdk_support_root}/matter/provision/lib/libProvision_si917.a" ]
```

Comment:
I suppose the family variable doesn't match the si917 name? Should we name the lib si91x?

Reply:
Only 1 family currently support on matter 917. The family name is currently set to `siwx917-common` which I don't like. I will change the set family name to si917 in a subsequent pr and it will match with the lib name. If other wifi family are added in the future we will see what name it will have/how we support them